app/client/components/user/CreateUserForm.tsx

```diff
@@ -72,7 +72,7 @@ class CreateUserForm extends React.Component<
   componentDidMount() {
     const { defaultDisbursement } = this.props;
     this.props.initialize({
-      accountType: TransferAccountTypes.USER.toLowerCase(),
+      accountTypes: [{ value: "USER", label: "USER" }],
```

Comment:
Ohhh this will be need to be changed with:
https://github.com/teamsempo/SempoBlockchain/pull/441/files#diff-8cca8a6af685d1a884e93491c72ced3bR2

Transfer account type of "User" shouldn't have been a valid option